src/mixins/filing-template-mixin.ts

```diff
@@ -187,21 +186,17 @@ export default class FilingTemplateMixin extends Vue {
   /**
    * Prepare name translations for non draft correction
    */
-  prepareNameTranslations () : NameTranslationIF {
-    const translations = this.getNameTranslations as NameTranslationDraftIF[]
-    return {
-      new: translations?.filter(x => x.action === ActionTypes.ADDED)
-        .map(x => x.value),
-      modified: translations?.filter(x => x.action === ActionTypes.EDITED)
-        .map(x => {
-          return {
-            newValue: x.value,
-            oldValue: x.oldValue
-          }
-        }),
-      ceased: translations?.filter(x => x.action === ActionTypes.REMOVED)
-        .map(x => x.value)
-    } as NameTranslationIF
+  prepareNameTranslations () : NameTranslationIF[] {
```

Comment:
I'd really like to see a unit test for the 3 methods you changed in this file.

Here's a test file for a mixin that you can copy:
https://github.com/bcgov/business-filings-ui/blob/master/tests/unit/common-mixin.spec.ts

Reply:
will look into it

Reply:
It may not be possible to include these in this ticket as we don't have an existing test file for this.

Reply:
Admittedly, we haven't created test files for most mixins so far, which is why it would be nice to catch up a little here for code you updated.
However, I don't want to slow you down too much, so if you have higher priority work for today then please create a tech-debt ticket for some unit tests for this mixin, that you can maybe get around to soon.

Reply:
I tried to include test cases for that. Then I realized to cover those methods I need to consider entire filing json. Which takes more time than what I have for this ticket. I was not pointing about not having test. I am just concerned about time :)